tock/tock/static/js/components/timecard.js

```diff
@@ -197,10 +197,10 @@ $("body").on("change", ".entry-project select", function () {
 
 
 $(document).ready(function () {
-  var chosenOptions = {
-    search_contains: true,
-    group_search: false
-  };
+  // var chosenOptions = {
+  // search_contains: true,
+  // group_search: false
+  // };
```

Comment:
✂️  I think we can probably delete these commented lines including the ones above, if that works with you!